service/routes.py

```diff
@@ -38,16 +38,90 @@ def healthcheck():
 @app.route("/")
 def index():
     """ Root URL response """
-    # return (
-    #     jsonify(
-    #         name="Orders REST API Service",
-    #         version="1.0",
-    #         paths=url_for("list_orders", _external=True),
-    #     ),
-    #     status.HTTP_200_OK,
-    # )
     return app.send_static_file("index.html")
 
+######################################################################
+# Configure Swagger before initializing it
+######################################################################
+api = Api(app,
+          version='1.0.0',
+          title='Orders REST API Service',
+          description='This is the back end for an eCommerce web site as a RESTful microservice for the resource order.',
+          default='orders',
+          default_label='Orders operations',
+          doc='/apidocs',
+          prefix='/api'
+          )
+
+# Define the model so that the docs reflect what can be sent
+create_item_model = api.model('Item', {
+    'item_id': fields.Integer(required=True,
+                                 description='Item id of the item'),
+    'item_name': fields.String(required=True,
+                            description='Name of the item'),
+    'item_qty': fields.Integer(required=True,
+                               description='Quantity of the item'),
+    'item_price': fields.Float(required=True,
+                          description='Price of the item')
+})
+
+item_model = api.inherit(
+    'ItemModel', 
+    create_item_model,
+    {
+        'id': fields.String(readOnly=True,
+                            description='The unique id assigned internally by service'),
+    }
+)
+create_model = api.model('Pet', {
+    'customer_id': fields.Integer(required=True,
+                                  description='The customer id of the Order'),
+    'order_items': fields.List(fields.Nested(create_item_model, required=True), required=True,
+                               description='The items in the Order')
+})
+
+order_model = api.inherit(
+    'OrderModel', 
+    create_model,
+    {
+        'id': fields.String(readOnly=True,
+                            description='The unique id assigned internally by service'),
```

Comment:
Lets make this description more explicit by saying something like : Unique order ID assigned internally by the service?